cartopy/documentation.html

```diff
@@ -53,14 +54,32 @@
           <h2>Previous releases</h2>
             <ul>
                 <li><a href="docs/v0.13/index.html">v0.13</a></li>
+            </ul>
```

Comment:
It looks like the iris page also does this. It seems like it is intended as a crude way to increase spacing between bullets. I'll accept either, as this will come down to personal preference. Consistency is important so we should make sure iris and cartopy documentation landing pages are using the same style.